Jenkinsfile

```
@@ -17,10 +17,27 @@ pipeline {
 
     stages {
         stage('Build Docker Image') {
-            steps {
-                deleteDir()
-                checkout scm
-                sh 'make build'
+            parallel {
+                stage('Windows') {
+                    agent {
+                        label 'windock'
+                    }
+                    steps {
+                        deleteDir()
+                        checkout scm
+                        powershell "& ./make.ps1"
```

```suggestion
                        powershell '& ./make.ps1'
```

As a matter of style, suggest always using single quotes unless you are actually trying to interpolate a Groovy variable…which is a bad idea in the case of `sh`/`bat`/`powershell` since it is safer and better to inject environment variables using `withEnv`.